src/DwmApi.Desktop/DwmApi+DWMWINDOWATTRIBUTE.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="DWMWINDOWATTRIBUTE"/> nested type.
+    /// </content>
+    public static partial class DwmApi
+    {
+        public enum DWMWINDOWATTRIBUTE : uint
+        {
+            NCRenderingEnabled = 1,
```

Comment:
Where constants are defined in native code, we want to [preserve the original names](https://github.com/AArnott/pinvoke/blob/master/CONTRIBUTING.md#naming) of the constants in enums. 
For example, this should be `DWMWA_NCRENDERING_ENABLED`.